v2/api/dbformysql/v1alpha1api20210501/flexible_server_types_gen.go

```diff
@@ -393,7 +393,11 @@ func (servers *FlexibleServers_Spec) ConvertToARM(resolved genruntime.ConvertToA
 		result.Properties.AdministratorLogin = &administratorLogin
 	}
 	if servers.AdministratorLoginPassword != nil {
-		administratorLoginPassword := *servers.AdministratorLoginPassword
+		administratorLoginPasswordSecret, err := resolved.ResolvedSecrets.SecretOrErr(*servers.AdministratorLoginPassword)
+		if err != nil {
+			return nil, err
```

Comment:
Suggest generating call here so that it's explicit which property was trying to use the secret, for example

```suggestion
			return nil, errors.Wrap(err, "looking up secret for AdministratorLoginPassword")
```

Reply:
Good idea.